classic.sh

```diff
@@ -24,33 +24,40 @@ case "$(uname -m)" in
         ;;
 esac
 
-if [ ! -f "xenial-base-$arch.tar.gz" ]; then
-    echo "Downloading xenial chroot for $arch..."
-    python3 -c "from urllib.request import urlretrieve; urlretrieve('http://cdimage.ubuntu.com/ubuntu-base/xenial/daily/current/xenial-base-$arch.tar.gz', 'xenial-base-$arch.tar.gz')"
+if [ ! -f "classic-base-$arch.tar.gz" ]; then
+    if [[ "$DEVEL" == "yes" ]]; then
+        echo "Downloading Ubuntu 16.10 (Development Branch) $arch Classic Base image..."
+        python3 -c "from urllib.request import urlretrieve; urlretrieve('http://cdimage.ubuntu.com/ubuntu-base/daily/current/yakkety-base-$arch.tar.gz', 'classic-base-$arch.tar.gz')"
+    else
+        echo "Downloading Ubuntu 16.04 LTS $arch Classic Base image..."
+	python3 -c "from urllib.request import urlretrieve; urlretrieve('http://cdimage.ubuntu.com/ubuntu-base/xenial/daily/current/xenial-base-$arch.tar.gz', 'classic-base-$arch.tar.gz')"
+    fi
 fi
 
-if [ ! -d xenial ]; then
-    mkdir xenial
-    echo "Uncompressing xenial chroot..."
-    tar -zxf "xenial-base-$arch.tar.gz" -C xenial
+if [ -f "classic-base-$arch.tar.gz" ]; then
```

Comment:
replace `xenial` with `$classic` everywhere.Then this script will let you pick-and-choose which classic chroot you want to use.